### ISSUE TITLE
v2.6.9 Unexpected content for method DELETE

### ISSUE THREAD

**Issue by FullofQuarks** [Telmate/terraform-provider-proxmox#338]
Provider version: 2.6.9
Terraform version: 0.15.1
PVE version: 6.4-4

Cannot delete an LXC during `terraform destroy`

The following is the relevant debug log chunk:

```
{"data":null}
2021/04/30 19:43:48 >>>>>>>>>> REQUEST:
DELETE /api2/json/nodes/pve1/lxc/115 HTTP/1.1
Host: 192.168.0.200:8006
User-Agent: Go-http-client/1.1
Content-Length: 2
Accept: application/json
Cookie: PVEAuthCookie=<redacted>
Accept-Encoding: gzip

""
2021/04/30 19:43:48 <<<<<<<<<< RESULT:
HTTP/1.1 501 Unexpected content for method 'DELETE'
Connection: close
Cache-Control: max-age=0
Date: Fri, 30 Apr 2021 19:43:48 GMT
Expires: Fri, 30 Apr 2021 19:43:48 GMT
Pragma: no-cache
Server: pve-api-daemon/3.0
```

**Comment by redzioch** [comment]
Same with VMs:

`2021/05/06 00:01:33 >>>>>>>>>> REQUEST:
DELETE /api2/json/nodes/pve2/qemu/101 HTTP/1.1
Host: 10.*.*.*:8006
User-Agent: Go-http-client/1.1
Content-Length: 2
Accept: application/json
Cookie: PVEAuthCookie=***
Csrfpreventiontoken: ***
Accept-Encoding: gzip

""
2021/05/06 00:01:33 <<<<<<<<<< RESULT:
HTTP/1.1 501 Unexpected content for method 'DELETE'
Connection: close
Cache-Control: max-age=0
Date: Wed, 05 May 2021 22:01:33 GMT
Expires: Wed, 05 May 2021 22:01:33 GMT
Pragma: no-cache
Server: pve-api-daemon/3.0
`

**Comment by tagur87** [comment]
I've noticed this also, I am wondering if it has to do with the timeout here not being long enough, going to do some testing.

https://github.com/Telmate/terraform-provider-proxmox/blob/master/proxmox/resource_vm_qemu.go#L1370

**Comment by sticky-note** [comment]
This is because the provider is sending `""` in the request.
By sending same request with curl with an empty body, it is working as expected
Maybe it is related to `proxmox-api-go`

**Comment by pgporada** [comment]
I'm experiencing this too.

**Comment by tagur87** [comment]
Should be fixed in #343 as it updates the latest api library to the patched one.

**Comment by mleone87** [comment]
I can confirm that latest api code  + latest PR manually merged works for my use cases
